lib/src/common/snow_3g.cc

```diff
@@ -1,515 +0,0 @@
-/*------------------------------------------------------------------------
- * snow_3g.c
- *
- * Adapted from ETSI/SAGE specifications:
```

Comment:
Should we perhaps add the ETSI reference to the new file?

Reply:
I included it but below the includes. Shall it be directly under the license header?